packages/@react-spectrum/list/src/listview.css

```
@@ -33,7 +36,8 @@
 
 .react-spectrum-ListViewItem {
   display: grid; /* TODO: define grid areas */
-  border-bottom: 1px solid var(--spectrum-tabel-cell-border-color, var(--spectrum-global-color-gray-300));
+  border: 1px solid transparent;
+  border-bottom-color: var(--spectrum-tabel-cell-border-color, var(--spectrum-global-color-gray-300));
```

```suggestion
  border-bottom-color: var(--spectrum-table-cell-border-color, var(--spectrum-global-color-gray-300));
```
Unless this was on purpose to force the gray fallback?

---

Nope, great catch. Definitely just a typo